jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/credentials/CodeWhispererClientAdaptor.kt

```diff
@@ -241,6 +241,7 @@ open class CodeWhispererClientAdaptorImpl(override val project: Project) : CodeW
                     it.timestamp(Instant.now())
                     it.suggestionReferenceCount(suggestionReferenceCount)
                     it.generatedLine(lineCount)
+                    it.customizationArn(requestContext.customizationArn)
```

Comment:
can we add a test to validate the existence of this field

Reply:
added

Reply:
for `modelConfigurator.activeCustomization()` it's tested by https://github.com/aws/aws-toolkit-jetbrains/blob/main/jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererModelConfiguratorTest.kt#L176